scripts/tsconfig.json

```diff
@@ -6,5 +6,5 @@
     "target": "ES2020"
   },
   "extends": "../tsconfig.json",
-  "include": ["**/*.ts", "**/*.d.ts"]
+  "include": ["**/*.ts", "**/*.d.ts", "**/*.liquid"]
```

Comment:
Do we still need this? I thought that TS wasn't moving the file, so we moved the `templates` directory up and just access it directly, not from `./dist`